src/main/java/org/myrobotlab/service/config/NeoPixelConfig.java

```diff
@@ -14,7 +14,7 @@ public class NeoPixelConfig extends ServiceConfig {
   public Integer brightness = 255;
   public boolean fill = false;
   // auto clears flashes
-  public boolean autoClear = true;
+  public boolean autoClear = false;
```

Comment:
why the change in default behavior here?

Reply:
also, it wasn't defined if "clear" was restore what was there before, or make blank